frontend/src/component/pages/shared/UserShortLinksSection.scss

```diff
@@ -11,7 +11,13 @@
     margin-top: 20px;
   }
 
-  .long-link {
+  .long-link,
+  .alias {
     word-break: break-all;
   }
+
+  td.cell-long,
+  th.cell-long {
```

Comment:
```suggestion
  .cell-long {
```

Reply:
Already done, but GitHub still thinks this is a change to be requested for some reason.